src/openrct2/paint/tile_element/Paint.SmallScenery.cpp

```diff
@@ -158,8 +158,8 @@ static void PaintSmallSceneryBody(
     else
     {
         uint8_t quadrant = (sceneryElement.GetSceneryQuadrant() + session.CurrentRotation) & 3;
-        offset.x = SceneryQuadrantOffsets[quadrant].x;
-        offset.y = SceneryQuadrantOffsets[quadrant].y;
+        offset.x = SceneryQuadrantOffsets[quadrant].x - 1;
```

Comment:
@Sadret Could you suggest which comment I should add?

Reply:
// -1 to maintain compatibility with existing CSOs in context of PR #17343

Reply:
or maybe 
// -1 to maintain compatibility with existing CSOs in context of issue #17616